java/ql/src/experimental/semmle/code/java/frameworks/JinJava.qll

```diff
@@ -0,0 +1,23 @@
+/* Definitions related to the Jinjava Tempalting library. */
+import semmle.code.java.Type
+
+/** Models `Jinjava` class of Jinjava Templating Engine. */
```

Comment:
For classes, document what it IS rather than what it DOES. For example, "The `Jinjava` class of Jinjava Templating Engine." not "Models..."